gpkit/model.py

```diff
@@ -176,7 +176,7 @@ def zero_lower_unbounded_variables(self):
                      if bound == "lower"}
             self.substitutions.update(zeros)
 
-    def solve(self, solver=None, verbosity=1, skipfailures=True,
+    def solve(self, solver=None, verbosity=1, skipfailures=False,
```

Comment:
Ah I see. Wonder if it should be called `skipsweepfailures`. But that's super minor, low priority, and unrelated to this PR. 

Thanks for explaining.